resources/assets/js/components/teams/TeamMembershipButton.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div>
+        <button
+                type="button" class="btn btn-secondary"
+                :disabled="!this.user.is_active || !this.team.self_serviceable"
+                v-if="memberOfTeam" v-on:click="changeMembership('leave')">
+            Leave
+        </button>
+        <button
+                type="button" class="btn btn-primary"
+                :disabled="!this.user.is_active || !this.team.self_serviceable"
+                v-else v-on:click="changeMembership('join')">
+            Join
+        </button>
+        <em v-if="!this.user.is_active">&nbsp;&nbsp;<small>You must be an active member to join/leave teams.</small></em>
```

Comment:
Do not display this message if the one below it also applies.

Also see my note about join/leave below.

Reply:
Note that the wording could be confusing if they then become active but still can't join/leave a non-SSE team.

Reply:
What I meant was you should _only_ display the SSE message if it applies, so if a member becomes active they still don't get a join/leave button.